app/blog/2017-07-12-an-update-on-neon-animation.md

```diff
@@ -0,0 +1,70 @@
+
+Animations are incredibly powerful, but thinking about animations through `neon-animations` limited the scope of what could be achieved.
+
+For these reasons, we're not planning on maintaining the package going forward.
+
+## What should I do?
+If you're creating a new element from scratch, we recommend using CSS keyframe animations, CSS transitions, or the Web Animations API.
+
+If you're using another element from the PolymerElements org that depends on `neon-animation`, like `iron-dropdown` or `paper-menu-button`, don't sweat it. We'll replace these animations.
+If you're using `neon-animation` directly, you should start planning to eventually move off it.
+
+## What about sharing animations?
+`neon-animation` gave the appearance of having a shared approach to animations, but in reality it provided a non-standard, ill-typed JSON structure that would then configure and fill in the parameters for the Web Animations API. This unfortunately locked you into our ill-defined animations API, which would then lock you into the technical debt of all the features of `neon-animations` that you were not using, and all we wanted was to apply a set of pre-defined parameters into the Web Animations API.
+
+What does it mean to share an animation? If the goal is to have a set of elements play the same animation in different parts of the code, then sharing an animation is the same as defining a set of keyframes and applying them to different elements. The question then becomes: how do you share a stylesheet or a javascript object / function across your project? There are many ways to do this: HTML imports, javascript modules, mixins, namespacing, XHRs to name a few. There is not a one-size-fits-all approach here: use what makes sense in your situation.
+
+Sharing is easy if you restrict your animation parameters to something like keyframes. If the goal is instead to share an animation concept with infinite flexibility, then you’d essentially be exposing the entire animations API and reinventing the wheel.
+
+## Alternatives
+As I've mentioned, animations have not gone away, they're still here in the platform. There are 3 different alternatives that we suggest (in no particular order):
+
+1. CSS Keyframe Animations
+2. Web Animations API
+3. CSS Transitions
+
+### CSS Keyframe Animations
+CSS Keyframe animations are great because they provide great control over animations. Ways they do this: they introduce the concept of a Keyframe which essentially represent a CSS snapshot at a specific percentage of an animation's timeline, CSS Keyframes also provide a nice, declarative syntax to define an animation by stylistically decoupling animations into the CSS. Another nice thing about CSS Keyframes is that they have been [supported](http://caniuse.com/#search=css%20animation) (prefixed) since Obama's first term (unprefixed around when [The Weeknd](https://www.youtube.com/watch?v=KEI4qSrkPAs) stopped feeling their face).
```

Comment:
I have mixed feelings about this. Kinda cute, but also kinda offputting for those of us who have no idea what The Weeknd is or why they stopped feeling their face. Not sure we should be rickrolling people from the blog. And the dates of Obama's first term may be a somewhat more hazy concept to users outside the US... (Although, with the state o' education today, maybe not...)